src/Widgets/SecretItemRow.vala

```diff
@@ -94,22 +94,15 @@ public class Wallet.SecretItemRow : Gtk.ListBoxRow {
             revealer.reveal_child = false;
 
             GLib.Timeout.add (revealer.transition_duration, () => {
-                delete_secret ();
+                var schema = new Secret.Schema (
+                    "io.elementary.switchboard.wallet", Secret.SchemaFlags.NONE,
+                    "brand", Secret.SchemaAttributeType.STRING,
+                    "exp", Secret.SchemaAttributeType.STRING
+                );
```

Comment:
This copies code from elsewhere - maybe should be held in a shared class and accessed with e.g. `WalletSchema.get_default ()` (?) Can be another PR though